packages/database/src/migrations/20201201082830-AddProjectPalauOlangch-modifies-data.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const COUNTRY_CODE = 'PW';
+const PROJECT_CODE = 'olangch_palau';
+const PROJECT_NAME = 'MoH Project Olangch';
+const DASHBOARDGROUP_NAME = 'General';
+const DASHBOARDGROUP_CODE = 'PW_Olangch';
+const DASHBOARDGROUP_COUNTRY_CODE = 'PW_Olangch_Country';
+const USER_GROUP = 'Donor';
+
+export const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+exports.up = async function (db) {
+  await insertObject(db, 'dashboardGroup', {
+    organisationLevel: 'Project',
+    userGroup: USER_GROUP,
+    organisationUnitCode: PROJECT_CODE,
+    dashboardReports: '{project_details}',
+    name: DASHBOARDGROUP_NAME,
+    code: DASHBOARDGROUP_CODE,
+    projectCodes: `{${PROJECT_CODE}}`,
+  });
+  await insertObject(db, 'dashboardGroup', {
+    organisationLevel: 'Country',
+    userGroup: USER_GROUP,
+    organisationUnitCode: COUNTRY_CODE,
+    dashboardReports: '{23,19,8,26}',
```

Comment:
@nick-beyondessential also another thing, I don't know if we'll want to show the above reports in this project? I know that we show them in `General` dashboard groups, but those are attached to the older projects of our platform (`explore`, `disaster`, `imms`, `fanafana`). To be honest, I'm not sure if we should show them in all those projects, perhaps we keep them because people are used into seeing them there.

I'd double check with Georgia whether `General` is the most appropriate name for this group, (perhaps it's not and we can suggest `Olangch` for now, as @Alastair-L pointed out). I would also confirm whether those reports here should be displayed in the project's dashboard

Reply:
Georgia is happy to follow convention. I have also removed those reports as they were just the ones in General and have used project_details until given specific reports.

Reply:
Sounds great, thanks!